psycop/common/sequence_models/train.py

```diff
@@ -54,10 +54,17 @@ def train(config_path: Path | None = None) -> None:
     trainer_kwargs = training_cfg.trainer.to_dict()
 
     # update config
+    std_logger.info("Updating Config")
     flat_config = flatten_nested_dict(config_dict)
     logger.experiment.config.update(flat_config)
 
-    # create dataloader:
+    # filter dataset
+    std_logger.info("Filtering Patients")
+    filter_fn = model.embedding_module.filter_and_reformat_events
+    training_dataset.filter_patients(filter_fn)
+    validation_dataset.filter_patients(filter_fn)
```

Comment:
We filter patient multiple times (both here and for the embedding module), but it is quite fast so I don't think it is a problem.